bench/concurrent_bench.ml

```diff
@@ -0,0 +1,19 @@
+open Main
+
+let read_index () =
+  let r = Index.v ~fresh:false ~readonly:true ~log_size index_name in
+  Fmt.epr "\n Read in readonly index. \n";
+  let count = ref 0 in
+  Index.iter
```

Comment:
does this work when the index is updated concurrently? e.g. will new written elements be read too?

Reply:
Not for readonly instances. Fixed by #81 .

Reply:
Oh I think I understand what you meant now.
The `iter` indeed won't see the elements added after it was called., so this bench will only read the elements that were there from previous benchs.